start

```diff
@@ -1,4 +1,4 @@
 mongod > /dev/null &
 PID="$!"
-./sbt run -mem 2048 -XX:MaxPermSize=512M -Dconfig.file=conf/application.conf
+./sbt run -mem 2048 -XX:MaxPermSize=512M -Dconfig.file=conf/application.conf -jvm-debug 5005
```

Comment:
Is this supposed to be changed on production?

Reply:
@jstriebel IIRC this script is not used for production, correct? If it is only local dev, then this should be fine.

Reply:
@jstriebel PING